pkg/hs/urbit-king/lib/Urbit/King/CLI.hs

```diff
@@ -77,6 +78,11 @@ data Nat
   | NatNever
   deriving (Show)
 
+data Injection
+  = InjectOneEvent   FilePath
+  | InjectManyEvents FilePath
+  deriving (Show)
```

Comment:
why is it necessary to support both of these?

Reply:
`-I` is a backwards compatibility concern with vere and there was a TODO to implement it, but it's also not exactly what you want for the use case we're using in production, where we want to give a sequence of events in a row for configuration reasons.

If we were starting from scratch, I wouldn't put `-I` in, but we aren't so I am.